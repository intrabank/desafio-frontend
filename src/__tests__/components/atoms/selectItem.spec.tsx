import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import SelectItem from '@/components/atoms/selectItem';

describe('SelectItem', () => {
	it('should render label component', () => {
		const tree = renderer.create(<SelectItem>Test</SelectItem>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
