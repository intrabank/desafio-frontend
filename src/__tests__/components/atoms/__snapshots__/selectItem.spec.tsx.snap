// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectItem should render label component 1`] = `
<option
  className="c-ixcDHV"
  disabled={false}
>
  Test
</option>
`;
